cmd/better-retester/retester.go

```diff
@@ -90,6 +82,31 @@ func findCandidates(config config.Getter, gc githubClient, usesGitHubAppsAuth bo
 	return prs, nil
 }
 
+func (c *retestController) atLeastOneRequiredJob(candidates map[string]tide.PullRequest) (map[string]tide.PullRequest, error) {
+	output := map[string]tide.PullRequest{}
+	for key, pr := range candidates {
+		// All
```

Comment:
// All presubmits that are required to pass for merging the PR.

Reply:
Yep will address (but tomorrow)